packages/database/src/migrations/1762221646418-createInvoicesInvoiceInsurancePlansTable.ts

```diff
@@ -0,0 +1,61 @@
+import { DataTypes, QueryInterface, Sequelize } from 'sequelize';
+
+const tableName = 'invoices_invoice_insurance_plans';
+
+const baseFields = {
+  id: {
+    type: DataTypes.TEXT,
+    defaultValue: Sequelize.fn('uuid_generate_v4'),
```

Comment:
This one should be preferred:
```suggestion
    defaultValue: Sequelize.fn('gen_random_uuid'),
```